docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
     extra_hosts:
       - 'host.docker.internal:host-gateway'
     ports:
-      - '${APP_PORT:-80}:80'
+      - '${APP_HOST:-127.0.0.1}:${APP_PORT:-80}:80'
```

Comment:
Can we move this to another PR.

Also, we'd need to repeat this to all containers, especially the database container to get the actual security benifit

Reply:
Good call. I'll make that change.